src/main/resources/db/changelog/changes/v1.1.0/sql/create-view-drv-inventory-statistical-code-full.sql

```diff
@@ -1,12 +1,11 @@
 DROP MATERIALIZED VIEW IF EXISTS drv_inventory_statistical_code_full;
+DROP MATERIALIZED VIEW IF EXISTS drv_circulation_loan_status;
+DROP MATERIALIZED VIEW IF EXISTS drv_inventory_item_status;
```

Comment:
I noticed that these 2 were still hanging around even though we don't use them. Could've put it in its own changeset but figured it'd be fine to just throw it in here.